book/src/tutorial/hardware.md

```diff
@@ -67,6 +67,10 @@ key.
 
 Once the one-time boot menu appears, select your USB drive and press enter.
 
+If this personally-built, unsigned binary fails to load,
+[secure boot] needs to be disabled.
```

Comment:
Let's move this warning up to the top of this section, and phrase it as a requirement. Something like: "Disable secure boot if it's enabled, so that an unsigned binary can be loaded."

Reply:
Thanks, done.

Reply:
On a meta note, how do you prefer updating pull requests? I amended my original commit and force pushed.